tools/sro2kmm/README.md

```diff
@@ -0,0 +1,61 @@
+## Overview
+
+`sro2kmm` is a shell script running an Ansible playbook in specific nodes of an existing OpenShift cluster in order to migrate the kernel modules loaded by [Special Resource Operator (SRO)](https://github.com/openshift/special-resource-operator) to the newer [Kernel Module Management operator (KMM)](https://github.com/rh-ecosystem-edge/kernel-module-management).
+
+## Requirements
+
+A running Openshift Cluster with:
+- Special Resource Operator installed and managing a DaemonSet which loads a kernel module.
+- Kernel Module Management operator installed and wanted Pods running the same kernel module name as in SRO workload.
+
+A computer with:
+
+- Installed [dialog](https://invisible-island.net/dialog/) package. Available in most distributions.
+- Openshift Client (`oc`) with access to OpenShift cluster.
+- SSH access to Openshift cluster nodes.
+- Ansible and Kubernetes python packages installed: `python3 -m pip install ansible kubernetes`
+(Tested on Ansible 2.13.7 and Python 3.8.15.). Further info at [Ansible Documentation](https://docs.ansible.com/ansible/latest/installation_guide/index.html).
+
+## Usage
+
+Before using sro2kmm you should modify `vars/ocp.yaml` to match your OpenShift Cluster credentials.
+Run `sro2kmm [sr_name] --dsmenu` script to begin the migration.
+
+There are three possible arguments to be used by the script. First is the name of the `SpecialResource` that you intend to migrate, which is a mandatory argument.
+Second argument is related to the settings applied to the kubernetes [k8s_drain](https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_drain_module.html#parameters) module, specifically the delete options used by it. Optional delete settings can be set adding argument `--dsettings` to the script which will show a checklist for the user to choose:
+
+- delete_emptydir_data
+Continue even if there are pods using emptyDir (local data that will be deleted when the node is drained).
+
+- disable_eviction
+Forces drain to use delete rather than evict.
+
+- force
+Continue even if there are pods not managed by a ReplicationController, Job, or DaemonSet.
+
+- ignore_daemonsets
+Ignore DaemonSet-managed pods.
```

Comment:
```suggestion
- `delete_emptydir_data`: continue even if there are pods using emptyDir (local data that will be deleted when the node is drained).
- `disable_eviction`: forces drain to use delete rather than evict.
- `force`: continue even if there are pods not managed by a ReplicationController, Job, or DaemonSet.
- `ignore_daemonsets`: ignore DaemonSet-managed pods.
```